modules/SurrealDB.yml

```diff
@@ -0,0 +1,16 @@
+name: SurrealDB
+description: A Nuxt module aimed to simplify the use of SurrealDB
+repo: sandros94/nuxt-surrealdb
+npm: nuxt-surrealdb
+icon: ''
+github: https://github.com/sandros94/nuxt-surrealdb
+website: https://github.com/sandros94/nuxt-surrealdb
+learn_more: ''
+category: Devtools
```

Comment:
When noticing that I didn't specify the `Database` category, I've also noticed that the bot changed it forcing `Devtools`